github-fetcher/Makefile

```diff
@@ -0,0 +1,4 @@
+all: test
+.PHONY: test
+test:
+	CGO_ENABLED=0 GOOS=linux go test ./...
```

Comment:
Why do you need `CGO_ENABLED=0` and why only test on linux? Shouldn't we just defer to the user's settings?